### DISCUSSION TITLE
Tasmota32 BLE + bluetooth button

### DISCUSSION THREAD

**Discussion by azslava** [arendst/Tasmota#22561] [Q&A]
Hi guys.
How do I connect the wireless button for the phone camera and assign it to turn on the relay? (bluetooth button)
[bluetooth button](https://www.amazon.com/Bluetooth-Smartphones-Wireless-Compatible-Cellphone/dp/B08L48WD4W/ref=sr_1_1?dib=eyJ2IjoiMSJ9.VZjLQ_Tp8jG2xwJ7XOUStEibsB1nmoOxJ8e0PznRVgd5rw3tQca3r_tHknsuZMxi4Q-IG9XrvVRYxobswOLdjpQ6YpKkoe86wXoX04aEkLKQ1RPkZ_3F5wI7TmcZC1eK0siKFonmj3o9hMjOoPnWVmCcuCZymq2n2uzk_5FfKY34X-iyP3eQR-VfH8R2LIhpqXZT4EoK2BHKg7iIiWku5ancUvkn5Xgrjay0oAMeG20.gn2EHLBonH4MEh28zPMCPGmupkTH_EnoiBbrAoYNOH8&dib_tag=se&keywords=bluetooth+button&qid=1732946862&sr=8-1)

**Comment by Staars** [comment]
This thing is not advertised as Bluetooth 4.x or higher. Thus chances are great, that it indeed does not support Bluetooth Low Energy, which means no chance with Tasmota.

**Reply by azslava** [reply]
I am interested in connecting similar buttons that work on Bluetooth 4.x

**Reply by Staars** [reply]
Then there is a chance, although this would need some Berry coding.
I expect these things to be HID-compatible and thus an active connection is needed, which per se is not ideal in the context of Tasmota (or any other IOT solution). For testing purposes I connected all kinds of BLE remotes to Tasmota and could get it it to work in every case (including microphone input, which of course is of very limited use). These are HID devices too.

My best guess is, that technically it will not be impossible, but you must be ready to tinker and invest some coding work for a adjusted Berry driver.
